js/components/report/iframe-answer.tsx

```diff
@@ -213,7 +214,7 @@ export class IframeAnswer extends PureComponent<IProps, IState> {
             {maybeAnswerTextOrLinks}
           </div>
         )}
-        {reportItemAnswerItems.map((item, index) => (
+        {!compareView && reportItemAnswerItems.map((item, index) => (
```

Comment:
Have you checked what happens with open response questions in this case?
As I wrote in the other PR. I think this will prevent open response questions from showing all of there features. It will also prevent something like SageModeler from showing its metrics in the compare view.